.github/ISSUE_TEMPLATE/user-story-issue.md

```diff
@@ -0,0 +1,23 @@
+---
+name: User Story
+about: This template provides a basic structure for user story issues.
+labels: Needs Triage
+---
+
+# Description
+As a ..., I want to ..., so I can ...
+
+<Link to Design Document or User Story>
+
+# Acceptance criteria
+
+- [ ] This is something that can be verified to show that this user story is satisfied.
```

Comment:
Similar comment as above:  It would be great to guide the type of ACs we want via the examples here.